### PR TITLE
Update T5, T9, T13, UCOB Korean translate

### PR DESCRIPTION
**Pull request by Una-p**
There are 2 untranslated part that require some logic at T9.
https://github.com/quisquous/cactbot/blob/39c5dbbb909420b1f61c36d681d6a8fddeb7f772/ui/raidboss/data/02-arr/raid/t9.js#L279
https://github.com/quisquous/cactbot/blob/39c5dbbb909420b1f61c36d681d6a8fddeb7f772/ui/raidboss/data/02-arr/raid/t9.js#L320-L325
And apologize, I cant test it right now and I'm afraid to gonna ruin this by working on it without testing. :(

### CODE REVIEW

**Comment by Una-p** [comment]
but it's tooo old thing to care about. It doesn't matter cause it can recognize the meaning.
If you don't mind, you can merge. 👍

**Comment by quisquous** [comment]
Yeah, if you want to refactor t9 to be translatable, go for it.

File: ui/raidboss/data/02-arr/raid/t9.js [hunk 1]
```diff
@@ -33,6 +34,8 @@
         de: 'Köder Supernova draußen',
         fr: 'Attirez les Supernovas à l\'extérieur',
         cn: '人群外放黑洞',
+        ko: '초신성 외곽으로 유도',
+
```
Comment (panicstevenson) [inline ui/raidboss/data/02-arr/raid/t9.js:38]: Extra enter here

File: ui/raidboss/data/02-arr/raid/t9.js [hunk 2]
```diff
@@ -435,7 +452,7 @@
         'Fireball': 'Feuerball',
         'Ghost': 'Geist',
         'Golem Meteors': 'Golem Meteore',
-        'Heavensfall': 'Himmelssturz',
+        'Heavensfall': 'Himmelsfall',
```
Comment (panicstevenson) [inline ui/raidboss/data/02-arr/raid/t9.js:455]: Was this intentional?

Comment (Una-p) [reply ui/raidboss/data/02-arr/raid/t9.js:455]: omg i'll open request again. thanks

Comment (quisquous) [reply ui/raidboss/data/02-arr/raid/t9.js:455]: I was about to do a release, so I'll fix these up.  No worries!

(Sorry for not noticing this myself.)

Comment (Una-p) [reply ui/raidboss/data/02-arr/raid/t9.js:455]: Umm already requested, how can I cancel it?..

**Pull request merged**